### PR TITLE
Fix deprecations in show.jl

### PR DESCRIPTION
**Pull request by Ayushk4**
Minor deprecations and bugs (similar to #153) fixed.

The function `summary` in show.jl ([source](https://github.com/JuliaText/TextAnalysis.jl/blob/3b046d2c205d1525b2acaf3c2f66b52ce69bfad5/src/show.jl#L17)) is named similar to [Base.summary](https://docs.julialang.org/en/v1/base/io-network/#Base.summary). 

@aviks  What should I rename this to? Or should I extend Base.summary to support  `AbstractDocument`, `Corpus` types?

### CODE REVIEW

File: src/show.jl [hunk 1]
```diff
@@ -1,9 +1,3 @@
-##############################################################################
-#
-# Basic printing of TextAnalysis types
-#
-##############################################################################
-
 show(io::IO, d::AbstractDocument) = print(io, "A $(typeof(d))")
```
Comment (Ayushk4) [inline src/show.jl]: Shouldn't this be Base.show?

**Comment by aviks** [comment]
This is good. Thanks @Ayushk4 

One comment is that the `show_repl` should be moved to `show(::IO, ::MIME"text/plain", c)`. See here for the current state of these functions: https://docs.julialang.org/en/v1/manual/types/#man-custom-pretty-printing-1

**Comment by aviks** [comment]
Thanks Ayush

**Comment by Ayushk4** [comment]
A good ( bad ? ) news - now that this is fixed. The outputs of print and show will be changed to the ones defined in `show.jl`.

This means the documentation work done in #134, #147, #150 needs to be revised for the new changes.

**Pull request merged**